packages/react-native/Libraries/Components/TextInput/InputAccessoryView.js

```diff
@@ -96,7 +101,11 @@ const InputAccessoryView: React.AbstractComponent<Props> = (props: Props) => {
         style={[props.style, styles.container]}
         nativeID={props.nativeID}
         backgroundColor={props.backgroundColor}>
-        {props.children}
+        <View style={{width}}>
+          <SafeAreaView style={styles.safeAreaView}>
```

Comment:
shouldn't these two be inverted? or, putting it in another way, can `SafeAreView` work as `View` (and we can remove one indentation level)?

Reply:
Yeah, makes sense! I just pushed a fix adding the width to the SafeArea itself and changed the commit message / PR title to something that matches what the PR does now.